chunky/src/java/se/llbit/chunky/ui/RenderCanvasFx.java

```diff
@@ -71,6 +72,8 @@
   private Vector2 target = new Vector2(0, 0);
   private Tooltip tooltip = new Tooltip();
 
+  private boolean fitToScreen = false;
```

Comment:
I'd say we should enable this by default. What do you think?

Reply:
Fixed bug in 431e63f
Truly set default in 047598f